### PR TITLE
[Readme] Update readme with recommended system requirements

### PR DESCRIPTION
**Pull request by srivickynesh**
This PR is opened to fix issue #26

Updated readme with minimum requirement and recommended requirements

Signed-off-by: Sri Vignesh <sselvan@redhat.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,6 +30,23 @@ System Requirements
     #. Device 2 (part of the topology)
     #. Device 3 (Optional additional device not part of the topology)
 
+Recommended System Requirements
+====================
```
Comment (vamahaja) [inline README.rst:34]: Allign this line with text

File: README.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ Prerequisites
 Create setup with below system requirements and install required packages on
 host from which this repo is used as library or tests are executed.
 
-System Requirements
+Minimum System Requirements
 ===================
```
Comment (vamahaja) [inline README.rst:20]: Allign this line with text

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (srivickynesh) [reply README.rst:20]: This is just a seperation for heading and body not actual decorator for heading.  This line wont be visible in readme.

File: README.rst (hunk 1 above)
Comment (srivickynesh) [reply README.rst:34]: This is just a seperation for heading and body not actual decorator for heading. This line wont be visible in readme.

**Review by vamahaja — APPROVED** [review]

**Review by SushilG96 — APPROVED** [review]
LGTM

**Pull request merged**